bindep.txt

```diff
@@ -8,7 +8,11 @@ libssh-dev [test platform:dpkg]
 libxml2-dev [test platform:ubuntu-bionic]
 libxslt1-dev [test platform:ubuntu-bionic]
 python2.7-dev [test platform:dpkg]
+python3.6-dev [test platform:ubuntu-bionic]
```

Comment:
What failure does this fix?  I assume something doesn't compile properly because python is missing.  Which version do we need on bionic?

Reply:
For now, I only create jobs for 3.6 and 3.8:

https://github.com/ansible/ansible-zuul-jobs/pull/451/files#diff-e0a9ae430091fa99c8f126c4410bc32dR32

Reply:
k, lets keep 3.6 for bionic, 3.7 can be dropped. We already installed 3.8 via ansible, based on ansible-test var.

Reply:
Roger that.